docs/spark-qualification-tool.md

```diff
@@ -384,6 +385,67 @@ For example, if running the spark-shell:
 $SPARK_HOME/bin/spark-shell --jars rapids-4-spark-tools_2.12-<version>.jar
 ```
 
+## Running using a Spark Listener
```

Comment:
@krajendrannv @viadea  let me know if this makes sense or we need more clarification.  Do we want to add any databricks specific documentation, user should just have to add configs but then needs init script to copy in tools jar.